inspirehep/records/ext.py

```diff
@@ -19,3 +19,4 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         app.extensions["inspirehep-records"] = self
+        from inspirehep.records.indexer.base import InspireRecordIndexer  # noqa: F401
```

Comment:
is this necessary, as far as I can see this file has only the class definition, nothing is executed when you import.

Reply:
Ouch, this is my mistake. I should import receivers here to register them properly.